### PR TITLE
Master

### PR DESCRIPTION
**Pull request by SergejKasperBM**
Enable easy regression-tests with grunt-galen. 

 Galens [dumpPage](http://galenframework.com/docs/reference-galen-javascript-api/#dumpPage) enables the automatic creation of base-images for all the objects of a page for a later visual comperison of those objects with the current state via galens [image](http://galenframework.com/docs/reference-galen-spec-language-guide/#Image).

### CODE REVIEW

**Comment by mjurczyk** [comment]
@SergejKasperBM sorry for a late response.

Can you fix the comments I added so that we can merge?

File: lib/gl.js [hunk 1]
```diff
@@ -399,6 +399,25 @@ gl = function () {
   };
 
   /**
+   * Dump a Galen spec file on the current page on a
+   * target driver.
+   *
+   * Argument `specFile` is relative to gl.js, not the test
+   * file.
+   *
```
Comment (mjurczyk) [inline lib/gl.js:407]: Please add for readability:

```
@see http://galenframework.com/docs/reference-galen-javascript-api/#dumpPage
```

File: lib/gl.js [hunk 2]
```diff
@@ -399,6 +399,25 @@ gl = function () {
...
+   * @param {Object} device   device specification
+   * @param {String} specFile Galen spec file path
+   */
+  function dumpSpecFile (device, specFile, dumpPath, dumpName) {
+    var useSingleDevice = config.getSingleDeviceState() && !config.getSeleniumGrid().enabled;
+    var specFileLevels = specFile.split("/");
+    var specFileName = dumpName || specFileLevels[specFileLevels.length -1];
```
Comment (mjurczyk) [inline lib/gl.js]: Please use spaces in math expressions (before 1):

```
var specFileName = dumpName || specFileLevels[specFileLevels.length - 1];
```

**Comment by mjurczyk** [comment]
@SergejKasperBM sorry for a late response, I didn't get a notification :(

Can you fix the comments I added so that we can merge?

Thank you ^_^

**Comment by SergejKasperBM** [comment]
Sorry for the late fix! Have been on vacation under complete codelessness _._ 
Hope that's good now. Let me know if there is anything else :)

**Comment by mjurczyk** [comment]
@SergejKasperBM not a problem, thanks ;) Please note that this module right now unfortunately becomes deprecated, since Galen v2.0.3+.

**Pull request merged**